test/e2e/benchmark/main.go

```diff
@@ -14,24 +14,18 @@ func main() {
 	}
 
 	// check the test name passed as an argument and run it
-	specificTestFound := false
-	for _, arg := range os.Args[1:] {
-		for _, test := range tests {
-			if test.Name == arg {
-				runTest(logger, test)
-				specificTestFound = true
-				break
-			}
-		}
-	}
-
-	if !specificTestFound {
-		logger.Println("No particular test specified. Running all tests.")
-		logger.Println("go run ./test/e2e/benchmark <test_name> to run a specific test")
+	if len(os.Args) < 2 {
+		logger.Println("No test is specified.")
+		logger.Println("Usage: go run ./test/e2e/benchmark <test_name>")
 		logger.Printf("Valid tests are: %s\n\n", getTestNames(tests))
-		// if no specific test is passed, run all tests
-		for _, test := range tests {
+		return
+
+	}
+	testName := os.Args[1]
+	for _, test := range tests {
+		if test.Name == testName {
 			runTest(logger, test)
+			break
 		}
 	}
```

Comment:
nit: we could add an error message here (and list all the possible benchmarks) in case the user doesn't specify a valid test

Reply:
Incorporated https://github.com/celestiaorg/celestia-app/pull/3618/commits/0722825e25b478fa9b726d65e411ffbffc9b1bfc